rolling-update-kubernetes/src/main/scala/akka/rollingupdate/kubernetes/CostStrategy.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2017-2021 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.rollingupdate.kubernetes
+
+import akka.annotation.InternalApi
+import akka.cluster.Member
+
+import scala.collection.SortedSet
+
+/**
+ *  INTERNAL API
+ *  Defines a trait for calculating the cost of removing a member from the akka cluster,
+ *  given said member and the list of the members of the cluster from oldest to newest.
+ */
+@InternalApi private[akka] trait CostStrategy {
+  def costOf(member: Member, membersByAgeDesc: SortedSet[Member]): Option[Int]
+}
+
+object OlderCostsMore extends CostStrategy {
+  def costOf(member: Member, membersByAgeDesc: SortedSet[Member]): Option[Int] = {
+    val nrMembersToAnnotate = 2
```

Comment:
config property, and let's use 3 as default

Reply:
thanks, done in https://github.com/akka/akka-management/pull/1113/commits/e037463e5789c85bc091750b6e3590800a74c968